docs/configuration/template_reference.md

```diff
@@ -48,6 +48,8 @@ If functions are used in a pipeline, the pipeline value is passed as the last ar
 | sortByLabel   | label, []samples | []sample | Sorts the samples by the given label. Is stable.  |
 
 `first`, `label` and `value` are intended to make query results easily usable in pipelines.
+The `value` variable holds the evaluated value of an alert. When the alarm is restored,
```

Comment:
This doesn't make sense, value is a function not a variable. This page is also about templates, not how alerts work.

Reply:
Yes, the value is a function in prometheus template, but user use it as a variable in alert rules, It is right?

Reply:
This is still not correct, as this section is also about templating - not alert fundamentals. This section already describes how $value works, so doesn't need changing.